src/uct/ib/mlx5/gdaki/gdaki.c

```diff
@@ -84,12 +85,62 @@ static void uct_rc_gdaki_calc_dev_ep_layout(size_t num_channels,
     *dev_ep_size_p       = qp_attr->umem_offset + qp_attr->len * num_channels;
 }
 
+static int uct_gdaki_is_dmabuf_supported(const uct_ib_md_t *md)
+{
+    static int dmabuf_supported = -1;
```

Comment:
so we really need to cache this value globalle, given that uct_cuda_copy_md_is_dmabuf_supported already caches the value?